screen/edit-transaction/src/main/java/com/ivy/transaction/EditTransactionViewModel.kt

```diff
@@ -598,7 +598,7 @@ class EditTransactionViewModel @Inject constructor(
     private fun createCategory(data: CreateCategoryData) {
         viewModelScope.launch {
             categoryCreator.createCategory(data) {
-                categories.value = categoryRepository.findAll().toImmutableList()
+                categories.value = categoryRepository.findAll().sortedBy { it.name.value }.toImmutableList()
```

Comment:
This shouldn't be the default behavior. Most users prefer to see the categories in the order that they've put them in the Categories screen. Make this feature a Setting using `Features` and IvyFeatures and by default it must be off